### PR TITLE
[AArch64] Minor improves to the vertex loader JIT

### PR DESCRIPTION
**Pull request by Sonicadvance1**
Just some minor improvements noticed by dumping the vertex loader blocks.

### CODE REVIEW

File: Source/Core/VideoCommon/VertexLoaderARM64.cpp [hunk 1]
```diff
@@ -347,12 +367,33 @@ void VertexLoaderARM64::GenerateVertexLoader()
...
+
+	bool has_tc = false;
+	bool has_tc_scale = false;
+	for (int i = 0; i < 8; i++)
+	{
+		has_tc |= tc[i];
+		has_tc_scale |= !!m_VtxAttr.texCoord[i].Frac;
+	}
+
+	bool need_scale = (m_VtxAttr.ByteDequant && m_VtxAttr.PosFrac) ||
```
Comment (degasus) [inline Source/Core/VideoCommon/VertexLoaderARM64.cpp]: Aren't this scale factors also for other attributes like texture coordinates?

**Comment by degasus** [comment]
LGTM but this one comment

File: Source/Core/VideoCommon/VertexLoaderARM64.cpp (hunk 1 above)
Comment (Sonicadvance1) [inline Source/Core/VideoCommon/VertexLoaderARM64.cpp]: Aye, which is why the next line has the check for if the texture coordinates are using the scale

Comment (degasus) [inline Source/Core/VideoCommon/VertexLoaderARM64.cpp]: oh, sorry ...

**Pull request merged**